go/desktop/web.config

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <system.webServer>
+        <rewrite>
+            <rules>
+                <clear />
+
+                <!-- Links for Desktop 10.0 onward -->
+
+                <rule name="/go/desktop/X.0/download-keyboards" stopProcessing="true">
+                  <match url="^(10|11|12|13)\.0/download-keyboards" />
```

Comment:
Let's make the match on `([1-9][0-9])\.[0-9]` so we don't need to update this every release.